consensus/safety-rules/src/test_utils.rs

```diff
@@ -200,7 +201,7 @@ pub fn make_timeout_cert(
 ) -> TwoChainTimeoutCertificate {
     let timeout = TwoChainTimeout::new(1, round, hqc.clone());
     let mut tc_partial = TwoChainTimeoutWithPartialSignatures::new(timeout.clone());
-    let signature = timeout.sign(signer);
+    let signature = timeout.sign(signer).unwrap();
```

Comment:
Can you mark this `make_timeout_cert` function as test-only via `#[cfg(test)]` to ensure it's not called anywhere else and panic?

Reply:
This is already covered by the module